pkg/deployer/terraform/deployer.go

```diff
@@ -239,6 +241,8 @@ func (d Deployer) createK8sJob(ctx context.Context, opts createK8sJobOptions) er
 		return err
 	}
 
+	jobEnv = d.addGitEnv(ctx, jobEnv)
+
```

Comment:
can we merge the `getProxyEnv` and `addGitEnv` together? like `getEnv`?

Reply:
updated